main.tex

```diff
@@ -15,35 +15,33 @@
 \maketitle
 
 \begin{abstract}
+  Bitcoin's design is based on pseudonymous identities, and a publicly verifiable ledger of transactions transferring value between them. Although this design lacks strong guarantees for privacy, it does not rule it out.
```

Comment:
> Bitcoin's design is based on pseudonymous identities

IMO this is a vast oversimplification, and doesn't stand for scrutiny.

Reply:
what do you think of this?

>   Bitcoin's transaction model is a publicly verifiable ledger of transferring value between pseudonymous identities. Although this design lacks strong guarantees for privacy, it does not rule it out.

shorter and more precise (doesn't make overly general statements), without actually adding unecessary details into the abstract

Reply:
My point was the word "pseudonymity" is red herring and we shouldn't use that to make blanket statements about Bitcoin. In fact it was a red herring from the very beginning: https://satoshi.nakamotoinstitute.org/emails/cryptography/threads/1/

> (Satoshi Nakamoto) It's not pseudonymous in the sense of nyms identifying people, but it is at least a little pseudonymous in that the next action on a coin can be identified as being from the owner of that coin.

Reply:
> Bitcoin's model for value transfer is publicly verifiable ledger of transactions, with ownership of coins defined in terms of public keys.

how's this?